### ISSUE TITLE
[api-extractor] dtsRollup does not emit an import for a symbol referenced using "typeof"

### ISSUE THREAD

**Issue by alan-agius4** [Microsoft/web-build-tools#1048]
When enabling `dtsRollup` imports for `typeof` references are lost.

**Input**
```ts
import { OperationCanceledException } from 'typescript';
export declare type Foo = typeof OperationCanceledException;
```

**Output**
```ts
export declare type Foo = typeof OperationCanceledException;
export { }
```

version: 7.0.11

**Comment by octogonz** [comment]
My theory is that `typeof` is a different AST node type that is not matched by [this switch statement](https://github.com/Microsoft/web-build-tools/blob/0497bbbd3aa5a855dcb830eae347e18596017562/apps/api-extractor/src/analyzer/AstSymbolTable.ts#L175).  Thus the analyzer doesn't crawl it.  I haven't investigated it yet, though.

**Comment by octogonz** [comment]
Fixed in PR https://github.com/Microsoft/web-build-tools/pull/1091

**Comment by octogonz** [comment]
This should be fixed in API Extractor 7.0.17 beta.  If not, please reopen this issue.

To install the beta, use this command:

```
npm install -g @microsoft/api-extractor@beta
```

(The "beta" releases have regular ordinary numbers like 7.0.17.  Our [NPM disttag](https://www.npmjs.com/package/@microsoft/api-extractor?activeTab=versions) prevents them from getting installed as the "latest" version if you simply run `npm install -g @microsoft/api-extractor`.  We will update the disttag as soon as version 7 is stable.)

**Comment by alan-agius4** [comment]
@octogonz, thanks confirmed that it works.
